app/src/main/java/com/github/multimatum_team/multimatum/MainActivity.kt

```diff
@@ -71,10 +74,14 @@ class MainActivity : AppCompatActivity() {
     Set a notification that will be triggered in a given time in ms.
     you can pass a title/description and Id in parameter
     */
-    private fun setNotification(timeMS: Long, title: String, description: String, id: Int){
-        alarmManager = getSystemService(ALARM_SERVICE) as AlarmManager  //this get an service instance of AlarmManager
-        val intent = Intent(this, ReminderBroadcastReceiver::class.java) //this create an intent of broadcast receiver
-        //Adding extra parameter that will be used in the broadcase receiver to create the notification
+    private fun setNotification(timeMS: Long, title: String, description: String, id: Int) {
+        alarmManager =
+            getSystemService(ALARM_SERVICE) as AlarmManager  //this get an service instance of AlarmManager
+        val intent = Intent(
+            this,
+            ReminderBroadcastReceiver::class.java
+        ) //this create an intent of broadcast receiver
+        //Adding extra parameter that will be used in the broadcast receiver to create the notification
         intent.putExtra("title", title)
         intent.putExtra("description", description)
         intent.putExtra("id", id)
```

Comment:
The keys should be constants.

Reply:
This is not my part, these were modified because of the auto-reformatting of android Studio